### PR TITLE
Add 5.15 release notes

### LINKED ISSUE #1864: Create 5.15 release notes
Will need release notes for 5.15

### PR DESCRIPTION
**Pull request by hillaryfraley**
## Description
Update the release notes page for 5.15

## Motivation and Context
Closes https://github.com/sensu/sensu-docs/issues/1864

### CODE REVIEW

File: content/sensu-go/5.15/release-notes.md [hunk 1]
```diff
@@ -7,6 +7,8 @@ version: "5.15"
 menu: "sensu-go-5.15"
 ---
 
+- [5.15.0 release notes](#5-15-0-release-notes)
+- [5.14.1 release notes](#5-14-1-release-notes)
```
Comment (nikkictl) [inline content/sensu-go/5.15/release-notes.md]: Did we not have release notes for 5.14.2?

Comment (hillaryfraley) [reply content/sensu-go/5.15/release-notes.md]: Yes, I need to add them. Thank you for catching it.

File: content/sensu-go/5.15/release-notes.md [hunk 2]
```diff
@@ -7,6 +7,8 @@ version: "5.15"
 menu: "sensu-go-5.15"
 ---
 
+- [5.15.0 release notes](#5-15-0-release-notes)
```
Comment (hillaryfraley) [inline content/sensu-go/5.15/release-notes.md:10]:
```suggestion
- [5.15.0 release notes](#5-15-0-release-notes)
- [5.14.2 release notes](#5-14-2-release-notes)
```

File: content/sensu-go/5.15/release-notes.md [hunk 3]
```diff
@@ -41,6 +43,60 @@ Read the [upgrade guide][1] for information on upgrading to the latest version o
...
+
+- You can now use colons and periods in all resource  and asset names (except users).
+
+**FIXED:**
+
+- Added better error logging for mutator execution.
+- Fixed the order of flap detection weighting for checks.
+- Fixed the pprof server so it only binds to localhost.
+- Moved `corev2.BonsaiAsset` to `bonsai.Asset` and moved `corev2.OutdatedBonsaiAsset` to `bonsai.OutdatedAsset`.
+
```
Comment (hillaryfraley) [inline content/sensu-go/5.15/release-notes.md:83]:
```suggestion

## 5.14.2 release notes

**November 4, 2019** &mdash; The latest release of Sensu Go, version 5.14.2, is now available for download. This release includes an etcd upgrade, fixes that improve stability and performance, and a Sensu Go package for CentOS 8.

See the [upgrade guide][1] to upgrade Sensu to version 5.14.2.

**IMPROVEMENTS:**

- Upgraded etcd to 3.3.17.
- Added build package for CentOS 8 (`el/8`).
- Sensu Go now uses serializable event reads, which helps improve performance.

**FIXES:**

- As a result of upgrading etcd, TLS etcd clients that lose their connection will successfully reconnect when using `--no-embed-etcd`.
- Check TTL and keepalive switches are now correctly buried when associated events and entities are deleted. As a result, Sensu now uses far fewer leases for check TTLs and keepalives, which improves stability for most deployments.
- Corrected a minor UX issue in interactive filter commands in sensuctl.


```

File: content/sensu-go/5.15/release-notes.md [hunk 4]
```diff
@@ -41,6 +44,79 @@ Read the [upgrade guide][1] for information on upgrading to the latest version o
...
+- Added support for using [API keys for API authentication][88].
+- Added support for sensu-backend service environment variables.
+
+**SECURITY**
+
+- ([Licensed][79]) Removed support for UPN binding without a binding account or anonymous binding, which allows Sensu to effectively refresh claims during access token renewal.
+
+**IMPROVEMENTS**
+
+- You can now use colons and periods in all resource  and asset names (except users).
```
Comment (nikkictl) [inline content/sensu-go/5.15/release-notes.md]:
```suggestion
- You can now use colons and periods in all resource names (except users).
```

**Review by nikkictl — COMMENTED** [review]
I'm going to hold off on approving this until all changes for 5.15.0 have landed, since the release notes will need to be updated with those additions.

Are we going to be making the change from "Licensed" to "Commercial" in the release notes, similar to what was done in https://github.com/sensu/sensu-docs/pull/1870?

File: content/sensu-go/5.15/release-notes.md [hunk 5]
```diff
@@ -41,6 +44,79 @@ Read the [upgrade guide][1] for information on upgrading to the latest version o
...
+
+**SECURITY**
+
+- ([Commercial feature][79]) Removed support for UPN binding without a binding account or anonymous binding, which allows Sensu to effectively refresh claims during access token renewal.
+
+**IMPROVEMENTS**
+
+- You can now use colons and periods in all resource names (except users).
+
+**FIXED:**
```
Comment (nikkictl) [inline content/sensu-go/5.15/release-notes.md]: `FIXES` is used consistently in previous releases.

File: content/sensu-go/5.15/release-notes.md [hunk 6]
```diff
@@ -41,6 +44,79 @@ Read the [upgrade guide][1] for information on upgrading to the latest version o
 
 ---
 
+## 5.15.0 release notes
+
+**November 18, 2019** &mdash; The latest release of Sensu Go, version 5.15.0, is now available for download. This is our biggest release yet and includes an important update to our free entity limit. We added the federation cluster registration API as well as the APIKey resource, which you can use for API authentication. In addition, the 5.15.0 release includes support for sensu-backend environment variables and bug fixes that improve error logging for mutator execution and flap detection weighting for checks.
+
+See the [upgrade guide][1] to upgrade Sensu to version 5.15.0.
+
+**IMPORTANT**
```
Comment (nikkictl) [inline content/sensu-go/5.15/release-notes.md]: Colons are used after sections names in previous releases.

File: content/sensu-go/5.15/release-notes.md [hunk 7]
```diff
@@ -41,6 +44,79 @@ Read the [upgrade guide][1] for information on upgrading to the latest version o
...
+
+If your Sensu instance includes more than 100 entities, [contact us][90] to learn how to upgrade your installation and increase your limit. See [Discourse][91] for more information about our usage policy.
+
+**NEW FEATURES:**
+
+- ([Commercial feature][79]) Added support for the [federation cluster registration API][85].
+- ([Commercial feature][79]) Added MSI and NuGet builds for [sensuctl][89]. Also, MSI and NuGet installations now add the bin directory to the system PATH on Windows.
+- ([Commercial feature][79]) Added HTTP DELETE access for the [license management API][92].
+- Added the [APIKey resource][86], with HTTP API support for POST, GET, and DELETE and [sensuctl commands][87] to manage the APIKey resource.
+- Added support for using [API keys for API authentication][88].
```
Comment (hillaryfraley) [inline content/sensu-go/5.15/release-notes.md:64]:
```suggestion
- Added support for using [API keys for API authentication][88].
- Added support for [sensuctl commands][93] to install, execute, list, and delete commands from Bonsai or a URL.
```

File: content/sensu-go/5.15/release-notes.md [hunk 8]
```diff
@@ -668,3 +744,12 @@ To get started with Sensu Go:
...
 [83]: /sensu-go/5.14/reference/backend/#advanced-configuration-options
+[84]: /sensu-go/5.14/sensuctl/reference/#exporting-resources
+[85]: /sensu-go/5.15/api/federation
+[86]: /sensu-go/5.15/reference/apikeys
+[87]: /sensu-go/5.15/guides/use-apikey-feature/#sensuctl-management-commands
+[88]: /sensu-go/5.15/api/overview/#authenticate-with-the-api-key-feature
+[89]: /sensu-go/5.15/sensuctl/reference/
+[90]: https://sensu.io/contact
+[91]: https://discourse.sensu.io/t/introducing-usage-limits-in-the-sensu-go-free-tier/1156
+[92]: /sensu-go/5.15/api/license/
```
Comment (hillaryfraley) [inline content/sensu-go/5.15/release-notes.md:757]:
```suggestion
[92]: /sensu-go/5.15/api/license/
[93]: /sensu-go/5.15/sensuctl/reference/#extend-sensuctl-with-commands
```

File: content/sensu-go/5.15/release-notes.md [hunk 9]
```diff
@@ -41,6 +44,80 @@ Read the [upgrade guide][1] for information on upgrading to the latest version o
...
+Sensu's free entity limit is now 100 entities. All [commercial features][79] are available for free in the packaged Sensu Go distribution up to an entity limit of 100. You will see a warning when you approach the 100-entity limit (at 75%).
+
+If your Sensu instance includes more than 100 entities, [contact us][90] to learn how to upgrade your installation and increase your limit. See [Discourse][91] for more information about our usage policy.
+
+**NEW FEATURES:**
+
+- ([Commercial feature][79]) Added support for the [federation cluster registration API][85].
+- ([Commercial feature][79]) Added MSI and NuGet builds for [sensuctl][89]. Also, MSI and NuGet installations now add the bin directory to the system PATH on Windows.
+- ([Commercial feature][79]) Added HTTP DELETE access for the [license management API][92].
+- Added the [APIKey resource][86], with HTTP API support for POST, GET, and DELETE and [sensuctl commands][87] to manage the APIKey resource.
```
Comment (hillaryfraley) [inline content/sensu-go/5.15/release-notes.md]:
```suggestion
- Added the [APIKey resource][86], with HTTP API support for POST, GET, and DELETE and [sensuctl APIKey resource commands][87] to manage the APIKey resource.
```

File: content/sensu-go/5.15/release-notes.md [hunk 10]
```diff
@@ -41,6 +44,80 @@ Read the [upgrade guide][1] for information on upgrading to the latest version o
...
+- Added support for using [API keys for API authentication][88].
+- Added support for [sensuctl commands][93] to install, execute, list, and delete commands from Bonsai or a URL.
+- Added support for sensu-backend service environment variables.
```
Comment (hillaryfraley) [inline content/sensu-go/5.15/release-notes.md:66]:
```suggestion
- Added support for sensu-backend service environment variables.
- Added support for [timezones in check `cron` strings][94].
```

File: content/sensu-go/5.15/release-notes.md [hunk 11]
```diff
@@ -668,3 +745,13 @@ To get started with Sensu Go:
...
+[84]: /sensu-go/5.14/sensuctl/reference/#exporting-resources
+[85]: /sensu-go/5.15/api/federation
+[86]: /sensu-go/5.15/reference/apikeys
+[87]: /sensu-go/5.15/guides/use-apikey-feature/#sensuctl-management-commands
+[88]: /sensu-go/5.15/api/overview/#authenticate-with-the-api-key-feature
+[89]: /sensu-go/5.15/sensuctl/reference/
+[90]: https://sensu.io/contact
+[91]: https://discourse.sensu.io/t/introducing-usage-limits-in-the-sensu-go-free-tier/1156
+[92]: /sensu-go/5.15/api/license/
+[93]: /sensu-go/5.15/sensuctl/reference/#extend-sensuctl-with-commands
```
Comment (hillaryfraley) [inline content/sensu-go/5.15/release-notes.md:758]:
```suggestion
[93]: /sensu-go/5.15/sensuctl/reference/#extend-sensuctl-with-commands
[94]: /sensu-go/5.15/reference/checks/#cron-scheduling
```

File: content/sensu-go/5.15/release-notes.md [hunk 12]
```diff
@@ -41,6 +44,81 @@ Read the [upgrade guide][1] for information on upgrading to the latest version o
...
+**NEW FEATURES:**
+
+- ([Commercial feature][79]) Added support for the [federation cluster registration API][85].
+- ([Commercial feature][79]) Added MSI and NuGet builds for [sensuctl][89]. Also, MSI and NuGet installations now add the bin directory to the system PATH on Windows.
+- ([Commercial feature][79]) Added HTTP DELETE access for the [license management API][92].
+- Added the [APIKey resource][86], with HTTP API support for POST, GET, and DELETE and [sensuctl commands][87] to manage the APIKey resource.
+- Added support for using [API keys for API authentication][88].
+- Added support for [sensuctl commands][93] to install, execute, list, and delete commands from Bonsai or a URL.
+- Added support for sensu-backend service environment variables.
+- Added support for [timezones in check `cron` strings][94].
```
Comment (nikkictl) [inline content/sensu-go/5.15/release-notes.md:67]: See https://github.com/sensu/sensu-go/commit/a93b17202ef29d7ffa29e212000aa148938ea541
```suggestion
- Added support for [timezones in check `cron` strings][94].
- Added support for extending sensuctl support with commands.
```

File: content/sensu-go/5.15/release-notes.md [hunk 13]
```diff
@@ -41,6 +44,81 @@ Read the [upgrade guide][1] for information on upgrading to the latest version o
...
+See the [upgrade guide][1] to upgrade Sensu to version 5.15.0.
+
+**IMPORTANT:**
+Sensu's free entity limit is now 100 entities. All [commercial features][79] are available for free in the packaged Sensu Go distribution up to an entity limit of 100. You will see a warning when you approach the 100-entity limit (at 75%).
+
+If your Sensu instance includes more than 100 entities, [contact us][90] to learn how to upgrade your installation and increase your limit. See [Discourse][91] for more information about our usage policy.
+
+**NEW FEATURES:**
+
+- ([Commercial feature][79]) Added support for the [federation cluster registration API][85].
```
Comment (nikkictl) [inline content/sensu-go/5.15/release-notes.md]: There are a few other federation components that ship in 5.15.0 (see https://github.com/sensu/sensu-enterprise-go/blob/master/CHANGELOG.md#unreleased). The cluster registration API is just one of those things. We could either list all components, or roll them all up into a single federation feature.

File: content/sensu-go/5.15/release-notes.md [hunk 14]
```diff
@@ -41,6 +44,81 @@ Read the [upgrade guide][1] for information on upgrading to the latest version o
 
 ---
 
+## 5.15.0 release notes
+
+**November 18, 2019** &mdash; The latest release of Sensu Go, version 5.15.0, is now available for download. This is a significant release for a number of reasons. The first being the changes to licensing, making 100% of Sensu Go's commercial features available for free to all users, up to your first 100 entities! This release also includes the long awaited Cluster Federation features, supporting multi-cluster authentication, RBAC policy replication, and a single pane of glass for your Sensu monitoring data! We added support for API keys, making it easy to integrate with the Sensu API (you no longer need to manage JWTs!). In addition, the 5.15.0 release includes support for sensu-backend environment variables and bug fixes that improve error logging for mutator execution and flap detection weighting for checks.
```
Comment (hillaryfraley) [inline content/sensu-go/5.15/release-notes.md]:
```suggestion
**November 18, 2019** &mdash; The latest release of Sensu Go, version 5.15.0, is now available for download. This is a significant release for a number of reasons. The changes to licensing make 100% of Sensu Go's commercial features available for free to all users, up to your first 100 entities! This release also includes the long awaited cluster federation features, supporting multi-cluster authentication, RBAC policy replication, and a single pane of glass for your Sensu monitoring data! We added support for API keys, making it easy to integrate with the Sensu API (you no longer need to manage JWTs). In addition, the 5.15.0 release includes support for sensu-backend environment variables and bug fixes that improve error logging for mutator execution and flap detection weighting for checks.
```

File: content/sensu-go/5.15/release-notes.md (hunk 12 above)
Comment (hillaryfraley) [reply content/sensu-go/5.15/release-notes.md:67]: @nikkixdev I have that listed above:
`Added support for [sensuctl commands][93] to install, execute, list, and delete commands from Bonsai or a URL.`

File: content/sensu-go/5.15/release-notes.md [hunk 15]
```diff
@@ -41,6 +44,81 @@ Read the [upgrade guide][1] for information on upgrading to the latest version o
...
+See the [upgrade guide][1] to upgrade Sensu to version 5.15.0.
+
+**IMPORTANT:**
+Sensu's free entity limit is now 100 entities. All [commercial features][79] are available for free in the packaged Sensu Go distribution up to an entity limit of 100. You will see a warning when you approach the 100-entity limit (at 75%).
+
+If your Sensu instance includes more than 100 entities, [contact us][90] to learn how to upgrade your installation and increase your limit. See [Discourse][91] for more information about our usage policy.
+
+**NEW FEATURES:**
+
+- ([Commercial feature][79]) Added support for the [federation cluster registration API][85].
```
Comment (hillaryfraley) [inline content/sensu-go/5.15/release-notes.md]:
```suggestion
- ([Commercial feature][79]) Added support for [federation replicators and the federation cluster registration API][85] and the ability to view resources across clusters in the federation in the [web UI][80].
```

**Review by nikkictl — APPROVED** [review]

File: content/sensu-go/5.15/release-notes.md [hunk 16]
```diff
@@ -41,6 +44,81 @@ Read the [upgrade guide][1] for information on upgrading to the latest version o
 
 ---
 
+## 5.15.0 release notes
+
+**November 18, 2019** &mdash; The latest release of Sensu Go, version 5.15.0, is now available for download. This is a significant release for a number of reasons. The changes to licensing make 100% of Sensu Go's commercial features available for free to all users, up to your first 100 entities! This release also includes the long awaited cluster federation features, supporting multi-cluster authentication, RBAC policy replication, and a single pane of glass for your Sensu monitoring data! We added support for API keys, making it easy to integrate with the Sensu API (you no longer need to manage JWTs). In addition, the 5.15.0 release includes support for sensu-backend environment variables and bug fixes that improve error logging for mutator execution and flap detection weighting for checks.
```
Comment (hillaryfraley) [inline content/sensu-go/5.15/release-notes.md]:
```suggestion
**November 19, 2019** &mdash; The latest release of Sensu Go, version 5.15.0, is now available for download. This is a significant release for a number of reasons. The changes to licensing make 100% of Sensu Go's commercial features available for free to all users, up to your first 100 entities! This release also includes the long awaited cluster federation features, supporting multi-cluster authentication, RBAC policy replication, and a single pane of glass for your Sensu monitoring data! We added support for API keys, making it easy to integrate with the Sensu API (you no longer need to manage JWTs). In addition, the 5.15.0 release includes support for sensu-backend environment variables and bug fixes that improve error logging for mutator execution and flap detection weighting for checks.
```

**Pull request merged**